lib/routes/initial_walkthrough/initial_walkthrough.dart

```diff
@@ -51,88 +52,91 @@ class InitialWalkthroughPageState extends State<InitialWalkthroughPage>
     final texts = context.texts();
     final themeData = Theme.of(context);
 
-    return Scaffold(
-      key: _scaffoldKey,
-      body: Padding(
-        padding: const EdgeInsets.only(top: 24.0),
-        child: Stack(
-          children: [
-            Column(
-              children: [
-                Expanded(
-                  flex: 200,
-                  child: Container(),
-                ),
-                Expanded(
-                  flex: 171,
-                  child: AnimatedBuilder(
-                    animation: _animation!,
-                    builder: (BuildContext context, Widget? child) {
-                      String frame = _animation!.value.toString().padLeft(2, '0');
-                      return Image.asset(
-                        'src/animations/welcome/frame_${frame}_delay-0.04s.png',
-                        gaplessPlayback: true,
-                        fit: BoxFit.cover,
-                      );
-                    },
+    return AnnotatedRegion<SystemUiOverlayStyle>(
+      value: themeData.appBarTheme.systemOverlayStyle!,
+      child: Scaffold(
```

Comment:
We should wrap Scaffold's with Theme widgets that uses light theme, don't we?
```
Theme(
      data: breezLightTheme,
      child: Scaffold(...
```

Reply:
True, this has been fixed.